build.sbt

```diff
@@ -114,6 +115,18 @@ lazy val googleCloudPubSub = alpakkaProject(
   parallelExecution in Test := false
 )
 
+lazy val googleCloudPubSubGrpc = alpakkaProject(
+  "google-cloud-pub-sub-grpc",
+  "google.cloud.pubsub.grpc",
+  Dependencies.GooglePubSubGrpc,
+  akkaGrpcCodeGeneratorSettings ~= { _.filterNot(_ == "flat_package") },
+  akkaGrpcGeneratedSources := Seq(AkkaGrpc.Client),
+  akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Scala, AkkaGrpc.Java),
+  javaAgents += Dependencies.GooglePubSubGrpcAlpnAgent % "test",
```

Comment:
@raboof will the users of alpakka-google-pub-sub-grpc connector need to add `javaAgent` to their build manually?

Reply:
I'm not completely sure if the Agent is needed only for the server side or also for the client - did you try it and see it fail without?

Reply:
Without it client gets stuck after:
```
...
2018-08-24 13:37:54,819 DEBUG [ExampleApp-akka.actor.default-dispatcher-3] [i.g.n.s.io.netty.handler.ssl.OpenSsl]  Supported protocols (OpenSSL): [SSLv2Hello, SSLv3, TLSv1, TLSv1.1, TLSv1.2]
2018-08-24 13:37:54,819 DEBUG [ExampleApp-akka.actor.default-dispatcher-3] [i.g.n.s.io.netty.handler.ssl.OpenSsl]  Default cipher suites (OpenSSL): [TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384, TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256, TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256, TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA, TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA, TLS_RSA_WITH_AES_128_GCM_SHA256, TLS_RSA_WITH_AES_128_CBC_SHA, TLS_RSA_WITH_AES_256_CBC_SHA]
```